LICENSE

```diff
@@ -229,7 +229,7 @@ BSD 3-Clause
 ------------
 
 python/lib/py4j-*-src.zip
-python/pyspark/cloudpickle.py
+python/pyspark/cloudpickle/*.py
```

Comment:
nit: is it only for cloudpickle.py and cloudpickle_fast.py?

Reply:
It caches the directory `python/pyspark/cloudpickle/` :-).

Reply:
Doesn't this use for license declaration?

Reply:
Oh

Reply:
I think it is not big deal to have BSD 3 for other two files. :)

Reply:
I think we ported `__init__.py` and `compat.py` here too and it includes all.

Looks like their license wasn't changed from the very first release (https://github.com/cloudpipe/cloudpickle/blob/master/LICENSE). I guess it might be fine..